packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.Props.ts

```diff
@@ -15,6 +15,11 @@ export interface IChoiceGroupProps extends React.HTMLProps<HTMLElement> {
    * Descriptive label for the choice group.
    */
   label?: string;
+
+  /**
+   * HTML Attribute for choice group options
+   */
+  choiceGroupName?: string;
```

Comment:
You don't even need to add name, since choiecgroupprops extends React.HtmlProps<htmlElement> it already has a name attribute.